from_cpython/setup.py

```diff
@@ -743,6 +743,10 @@ def detect_modules(self):
             missing.extend(['imageop'])
 
         # readline
+        # Pyston change: in some container system, the dependencies are provided by
```

Comment:
mmh I'm not so sure about this change, I don't think it hurts but I'm not sure and would prefer if you server would not need this.
I found cpythons https://bugs.python.org/issue21571 but it's still open :-(.
@kmod what do you think about this change?